src/openforms/submissions/migrations/0079_cleanup_urls.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.21 on 2023-11-16 14:57
+from django.db import migrations
+from django.db.migrations.state import StateApps
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+from furl import furl
+
+
+def cleanup_submission_urls(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+
+    Submission = apps.get_model("submissions", "Submission")
+
+    for submission in Submission.objects.all():
```

Comment:
```suggestion
    for submission in Submission.objects.iterator():
```
there may be many records and you want to avoid loading them all in memory